src/API/PaymentGatewaySuggestions.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * REST API Payment Gateway Suggestions Controller
+ *
+ * Handles requests to install and activate depedent plugins.
+ */
+
+namespace Automattic\WooCommerce\Admin\API;
+
+use Automattic\WooCommerce\Admin\Features\PaymentGatewaySuggestions\Init as Suggestions;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * PaymentGatewaySuggetsions Controller.
+ *
+ * @extends WC_REST_Data_Controller
+ */
+class PaymentGatewaySuggestions extends \WC_REST_Data_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-admin';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'payment-gateway-suggestions';
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_suggestions' ),
+					'permission_callback' => array( $this, 'get_permission_check' ),
+				),
+				'schema' => array( $this, 'get_item_schema' ),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/payment-gateway-suggestions/dismiss',
```

Comment:
This `'/payment-gateway-suggestions/dismiss'` should be `'/dismiss'` correct? Currently the dismiss action causes a 404 on the front-end.

Reply:
Good catch.  Updated and also optimistically dismissed the recommendations client-side to avoid the delay we were seeing.

There is a lot more renaming that could be done around "recommendations" and moving methods within data stores, but I'm going to try and keep the changes in this PR to as little as possible and we can follow-up on those.